tools/retis_in_container.sh

```diff
@@ -64,9 +63,10 @@ if [ -f ${OVS_RUNDIR:-/var/run/openvswitch}/ovs-vswitchd.pid ]; then
 fi
 
 # Run the Retis container.
-exec $runtime run $extra_args $term_opts --privileged --rm --pid=host \
+exec $runtime run $extra_args $term_opts --rm --pid=host \
+      --cap-drop all --security-opt no-new-privileges --read-only --net none \
+      --cap-add SYS_ADMIN --cap-add BPF --cap-add CAP_PERFMON --cap-add SYSLOG --cap-add DAC_OVERRIDE \
```

Comment:
For `--ovs-track` to work we need `SYS_PTRACE`, and for `--allow-system-changes` we need `CAP_NET_ADMIN`

Reply:
Thanks for testing! I forgot to test with those optional modes...

More generally, this is the tricky commit from this PR. I think there's value in restricting the container as much as we can, but that also comes with potential issues and it's not like it'll run without having wide rights. There's a balance between making it a bit more contained and usability, so feel free to push back on this.